doc/setup-binderhub.rst

```diff
@@ -177,6 +177,8 @@ Second, you will need to instruct BinderHub about the token URL::
       DockerRegistry:
         token_url: "https://myregistry.io/v2/token?service="
 
+If you are using basic HTTP authentication (htpasswd), you can set token_url to None.
```

Comment:
```suggestion
If you are using basic HTTP authentication (htpasswd), you can set ``token_url`` to ``None``.
```

Will people know why/if they are using basic HTTP auth? Maybe we can expand this a bit to say what you said in the PR description "small local deployments with ..." so that people have an easier time realising when this is appropriate to use.